web-ng/src/app/services/router/router.service.ts

```diff
@@ -164,8 +162,8 @@ export class RouterService {
   }
 }
 
-export enum SideNavContentType {
-  LAYER_LIST = 0,
-  OBSERVATION = 1,
-  FEATURE = 2,
+export enum SideNavContentMode {
+  LAYER_LIST = 1,
```

Comment:
Can we return this to 0, 1, 2 now?

Reply:
unfortunately still does not work with 0 (even with switch)

Reply:
Very odd.. I don't think `switch` would ignore falsey values, so something else is probably amiss. Let's leave it like this for now, but would good to understand why it's happening!